src/stores/spaces/SpaceStore.ts

```diff
@@ -387,6 +393,13 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
         return this.spaceFilteredRooms.get(space) || new Set();
     };
 
+    public getSpaceFilteredDirectChildRoomIds = (space: SpaceKey): Set<string> => {
+        if (space === MetaSpace.Home && this.allRoomsInHome) {
+            return this.getSpaceFilteredRoomIds(space);
+        }
```

Comment:
IMO all metaspaces should get handled by the non-direct variant given metaspaces can't have child spaces